onnxruntime/python/tools/microbench/benchmark.py

```diff
@@ -0,0 +1,49 @@
+import time
+import onnxruntime as ort
+import torch
+
+
+def create_io_binding(sess, inputs, outputs):
+    io_binding = sess.io_binding()
+    device = "cuda"
+
+    for name, array in inputs.items():
+      tensor = torch.from_numpy(array).to(device)
```

Comment:
since you rely on torch to create buffer on device, you can pass in torch tensor directly, instead of rely on numpy array.

Reply:
This is a good idea. I gave it a try, but it turns the current io binding interface requires numpy type: 
https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/onnxruntime_pybind_iobinding.cc#L121